packages/atlas/src/views/global/NftSettlementBottomDrawer/NftSettlementBottomDrawer.tsx

```diff
@@ -0,0 +1,107 @@
+  const { nft, loading, refetch } = useNft(currentNftId || '')
+
+  const lastBidder =
+    (nft?.transactionalStatus.__typename === 'TransactionalStatusAuction' &&
+      nft.transactionalStatus.auction?.lastBid?.bidder) ||
+    null
+
+  const { displaySnackbar } = useSnackbar()
+  const { isLoadingAsset: thumbnailLoading, url: thumbnailUrl } = useAsset(nft?.video.thumbnailPhoto)
+  const { url: avatarUrl } = useAsset(nft?.video.channel.avatarPhoto)
+  const { url: memberAvatarUrl } = useMemberAvatar(lastBidder)
+
+  const { joystream, proxyCallback } = useJoystream()
+  const handleTransaction = useTransaction()
+  const { activeMemberId } = useUser()
+
+  const handleSettleAuction = () => {
+    if (!joystream || !currentNftId || !activeMemberId) return
+
+    handleTransaction({
+      txFactory: async (updateStatus) =>
+        (await joystream.extrinsics).settleEnglishAuction(currentNftId, activeMemberId, proxyCallback(updateStatus)),
+      onTxSync: () => {
+        displaySnackbar({
+          title: 'Auction settled',
+          description: 'Your auction has been settled. You are now the owner of this NFT',
+          iconType: 'success',
+        })
+        closeNftAction()
+        return refetch()
+      },
+      successMessage: {
+        title: 'Auction settled',
+        description: 'Good job',
+      },
+    })
+  }
+  const isOpen = currentAction === 'settle'
+  return (
+    <BottomDrawer isOpen={isOpen} onClose={closeNftAction} coverTopbar>
+      <StyledLottie play={isOpen} loop={false} animationData={confetti} />
+      <StyledLimitedContainer>
+        <StyledLayoutGrid>
+          <GridItem rowStart={{ base: 2, sm: 1 }} colSpan={{ base: 12, sm: 6, md: 5, lg: 4 }} colStart={{ lg: 3 }}>
+            <NftCard
+              title={nft?.video.title}
+              thumbnail={{
+                loading: thumbnailLoading,
+                thumbnailUrl: thumbnailUrl,
+              }}
+              creator={{ name: nft?.video.channel.title, assetUrl: avatarUrl }}
+              owner={{ name: lastBidder?.handle, assetUrl: memberAvatarUrl }}
+              fullWidth
+              loading={loading}
+            />
+          </GridItem>
+          <StyledGridItem
+            rowStart={{ base: 1, sm: 1 }}
+            colSpan={{ base: 12, md: 6, lg: 4 }}
+            colStart={{ sm: 8, md: 7, lg: 8 }}
+          >
+            <Content>
+              <Text variant="h600">You have won the auction! 🎉</Text>
+              <Text variant="t300" secondary margin={{ top: 4, bottom: 10 }}>
+                Congratulations! To update the ownership, you need to settle the auction.
+              </Text>
+              <Button size="large" fullWidth={!xsMatch} onClick={handleSettleAuction}>
+                Settle the auction
+              </Button>
+              <Text variant="t100" secondary margin={{ top: 4 }}>
+                Transaction fee: <Text variant="t100">0 tJoy</Text>
```

Comment:
Let's try this again :D
```suggestion
                Transaction fee: <Text variant="t100">0 tJOY</Text>
```